Shokofin/Configuration/PluginConfiguration.cs

```diff
@@ -110,7 +110,7 @@ public PluginConfiguration()
             FilterOnLibraryTypes = false;
             UserList = Array.Empty<UserConfiguration>();
             IgnoredFileExtensions  = new [] { ".nfo", ".jpg", ".jpeg", ".png" };
-            IgnoredFolders = new [] { ".streams", "@Recently-Snapshot" };
+            IgnoredFolders = new [] { ".streams", "@recently-snapshot", "$recycle.bin", ".recycle.bin", "#recycle" };
```

Comment:
The recycle-bin paths are already in the core. And I think it would be better to add those missing in the core to the core.

https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Library/IgnorePatterns.cs

Reply:
Removed the folders from the list and might issue a PR on jellyfin as you said; makes sense to expand the list there